### PR TITLE
Document Editing Interfaces 

### PR DESCRIPTION
**Pull request by sdepold**

### CODE REVIEW

**Comment by sdepold** [comment]
@geigerzaehler Can you please add descriptions to the list of available widgets? I also wonder if we could format them differently and if it makes sense to list them all.

File: apiary/Makefile [hunk 1]
```diff
@@ -60,10 +60,7 @@ $(CPA_BLUEPRINT): outdir cpa.apib
 prepare: $(CDA_BLUEPRINT) $(CPA_BLUEPRINT) $(CMA_BLUEPRINT) $(IMAGES_BLUEPRINT)
 
 preview: check_apiary_token prepare
-	$(APIARY) preview --path=$(CDA_BLUEPRINT)
 	$(APIARY) preview --path=$(CMA_BLUEPRINT)
-	$(APIARY) preview --path=$(CPA_BLUEPRINT)
-	$(APIARY) preview --path=$(IMAGES_BLUEPRINT)
```
Comment (realityking) [inline apiary/Makefile]: I guess this wasn't intentional, was it?

Comment (sdepold) [inline apiary/Makefile]: iek. correct

Comment (sdepold) [inline apiary/Makefile]: fixed

**Comment by sdepold** [comment]
@geigerzaehler please file another PR with an update to the table

**Pull request merged**